packages/ember-glimmer/lib/renderer.js

```diff
@@ -137,6 +138,7 @@ class Renderer {
       controller,
       targetObject: controller,
       outletState: view.toReference(),
+      rootOutletState: view.toReference(),
```

Comment:
I **think** that means the tests are not testing deep enough nesting – if the __orphan__ thing is only available at the root, wouldn't it mean you need to got through that?